src/probabilistic.jl

```diff
@@ -544,3 +544,85 @@ $DOC_DISTRIBUTIONS
 SphericalScore
 "$SphericalScoreDoc"
 const spherical_score = SphericalScore()
+
+
+# ---------------------------------------------------------------------
+# Continuous Boyce Index
+struct _ContinuousBoyceIndex 
+    nbins::Integer
+    bin_overlap::AbstractFloat
+    min::Union{AbstractFloat, Nothing}
+    max::Union{AbstractFloat, Nothing}
+    cor::Function
+    function _ContinuousBoyceIndex(; nbins = 101, bin_overlap = 0.1, min = nothing, max = nothing, cor = StatsBase.corspearman)
+        new(nbins, bin_overlap, min, max, cor)
+    end
+end
+
+ContinuousBoyceIndex(; kw...) = _ContinuousBoyceIndex(; kw...) |> robust_measure |> fussy_measure
+
+function (m::_ContinuousBoyceIndex)(ŷ::UnivariateFiniteArray, y::NonMissingCatArrOrSub; warn=true)
+    warn && warn_unordered(levels(y))
+    positive_class = levels(first(ŷ))|> last
+    scores = pdf.(ŷ, positive_class)
+    max = isnothing(m.max) ? maximum(scores) : m.max
+    min = isnothing(m.min) ? minimum(scores) : m.min
+    binwidth = m.bin_overlap * (max - min)
+
+    return Functions.cbi(scores, y, positive_class; nbins = m.nbins, binwidth, max, min, cor = m.cor)
+end
+
+const ContinuousBoyceIndexType = API.FussyMeasure{<:API.RobustMeasure{<:_ContinuousBoyceIndex}}
+
+@fix_show ContinuousBoyceIndex::ContinuousBoyceIndexType
+
+StatisticalMeasures.@trait(
+    _ContinuousBoyceIndex,
+    consumes_multiple_observations=true,
+    observation_scitype = Finite{2},
+    kind_of_proxy=StatisticalMeasures.LearnAPI.Distribution(),
+    orientation=Score(),
+    external_aggregation_mode=Mean(),
+    human_name = "continuous boyce index",
```

Comment:
Is `Boyce` not a proper name?

```suggestion
    human_name = "continuous Boyce index",
```